core/src/main/java/gov/nasa/jpl/mudrod/discoveryengine/RecommendEngine.java

```diff
@@ -21,13 +21,11 @@
 
   public RecommendEngine(Properties props, ESDriver es, SparkDriver spark) {
     super(props, es, spark);
-    // TODO Auto-generated constructor stub
     LOG.info("Started Mudrod Recommend Engine.");
   }
 
   @Override
   public void preprocess() {
-    // TODO Auto-generated method stub
     LOG.info("*****************Recommendation preprocessing starts******************");
```

Comment:
We really ought to get rid of this logging. It is totally unnecessary. All we need is a simple log message and possibly a timing metric so we can determine, at a glance, how long each stage took to complete or fail.